lang/pt-BR/index.md

```diff
@@ -287,7 +286,7 @@ participação de:
 Toda colaboração na tradução pode ser acompanhada no link:
 http://pad.okfn.org/p/Fh9hjBPVu9
 
-Caso queira deixar sua opinião, por favor [abra uma issue no GitHub](https://github.com/wendtecnologia/semver/issues).
+Caso queira deixar sua opinião, por favor [abra uma issue no GitHub](https://github.com/mojombo/semver/issues).
```

Comment:
Should it be replaced with `https://github.com/semver/semver.org/issues`?

Reply:
@alexandrtovmach I don't know. Do you think that I should update this link?

Reply:
Based on original content we should use `semver/semver.org`:
https://github.com/semver/semver.org/commit/b5e2e75374611657c0c62e0bcc8d03172d49e3d5#diff-c2d2fbe9e5c48c67f69cc53d17e084daR357

If you'd like to update your PR with that - it'll be great 👍

Reply:
```suggestion
Caso queira deixar sua opinião, por favor [abra uma issue no GitHub](https://github.com/semver/semver.org/issues).
```